app/helpers/mailer_helper.rb

```diff
@@ -176,6 +191,85 @@ def style_work_tag_metadata(tags)
+  # otherwise pass invalid or anonymity-breaking combinations of data to this.
+  def batch_subscription_subject(subscription, creation, additional_creations_count)
+    subscribable_type = subscription.subscribable_type.downcase
+
+    creation_type = creation.is_a?(Chapter) ? "chapter" : "work"
+
+    work = creation.is_a?(Chapter) ? creation.work : creation
+    series = subscription.subscribable if subscribable_type == "series"
+    creator_list = creation.pseuds.map(&:byline).to_sentence unless creation.anonymous?
+    # For pluralization: creator publico, creator y creator2 publicaron.
+    creators_count = creation.pseuds.size unless creation.anonymous?
+    chapter_header = creation.chapter_header if creation_type == "chapter"
+
+    base_key = "user_mailer.batch_subscription_notification.subject"
+    creator_key = creation.anonymous? ? "anon" : "named"
+    creation_key = subscribable_type == "series" ? "series.#{creation_type}" : creation_type
+    entries_key = additional_creations_count.zero? ? "one_entry" : "multiple_entries"
+
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.anon.chapter.multiple_entries")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.anon.chapter.one_entry")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.anon.series.chapter.multiple_entries")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.anon.series.chapter.one_entry")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.anon.series.work.multiple_entries")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.anon.series.work.one_entry")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.named.chapter.multiple_entries")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.named.chapter.one_entry")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.named.series.chapter.multiple_entries")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.named.series.chapter.one_entry")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.named.series.work.multiple_entries")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.named.series.work.one_entry")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.named.work.multiple_entries")
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.named.work.one_entry")
+    computed_key = "#{base_key}.#{creator_key}.#{creation_key}.#{entries_key}"
+
+    # "and X more," translated separately so we can pluralize "more" based on X.
+    # i18n-tasks-use t("user_mailer.batch_subscription_notification.subject.more")
+    more_translation = t("#{base_key}.more", count: additional_creations_count) unless additional_creations_count.zero?
+
+    variables = {}
```

Comment:
Hmm, not a big fan of `variables` as a name here — it feels a little generic. `interpolations`, maybe? Or `translation_options`?